From 5aad10b49e193ee14a86e1277146a223005a2d6b Mon Sep 17 00:00:00 2001
From: Jakub Hrozek <jhrozek@redhat.com>
Date: Mon, 13 May 2013 10:23:56 +0200
Subject: [PATCH] Re-add a useful DEBUG message

In commit 46222e5191473f9a46aec581273eb2eef22e23be we removed a very
similar DEBUG message while moving the whole piece of code to the idmap
library. But it turned out that the DEBUG message was useful while
testing the functionality, so this patch adds it back.
---
 src/providers/ldap/sdap_idmap.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/providers/ldap/sdap_idmap.c b/src/providers/ldap/sdap_idmap.c
index 050b2c5a768f58acd376e68a15a579e1e56894ac..43db0c83354ed2f8f112a8853ab66ab51e1d9fd2 100644
--- a/src/providers/ldap/sdap_idmap.c
+++ b/src/providers/ldap/sdap_idmap.c
@@ -242,6 +242,8 @@ sdap_idmap_add_domain(struct sdap_idmap_ctx *idmap_ctx,
         ret = EIO;
         goto done;
     }
+    DEBUG(SSSDBG_TRACE_LIBS,
+          ("Adding domain [%s] as slice [%llu]\n", dom_sid, slice));
 
     if (range.max > idmap_upper) {
         /* This should never happen */
-- 
1.8.2.1

